DESCRIPTION

```diff
@@ -56,6 +56,7 @@ Suggests:
     rmarkdown,
     shinyvalidate,
     teal.code (>= 0.3.0.9009),
+    teal.modules.general,
```

Comment:
cc @chlebowa - I'm still unsure if `landing_popup` which is not a module (yet) should be a part of `tmg` or `teal.widgets` that rather hosts functions

Reply:
`teal.modules.general` `Depends` on `teal`.

Reply:
this part is outdated